src/seedu/addressbook/data/person/Person.java

```diff
@@ -58,6 +60,16 @@ public Address getAddress() {
     public UniqueTagList getTags() {
         return new UniqueTagList(tags);
     }
+    
+    public Tagging addTagging(Tag tag) {
+    	Tagging tagging = new Tagging(TaggingType.ADD, this, tag);
+    	return tagging;
+    }
+    
+    public Tagging deleteTagging(Tag tag) {
+    	Tagging tagging = new Tagging(TaggingType.DELETE, this, tag);
+    	return tagging;
+    }
```

Comment:
It's a little queer that `Person` should know about tagging.  Let `AddressBook` handle it.